lib/fs/basicfs_windows.go

```diff
@@ -150,3 +151,31 @@ func (f *BasicFilesystem) Roots() ([]string, error) {
 
 	return drives, nil
 }
+
+func (f *BasicFilesystem) resolveWin83(absPath string) string {
+	if !strings.Contains(absPath, "~") {
+		return absPath
+	}
+	if in, err := syscall.UTF16FromString(absPath); err != nil {
+		out := make([]uint16, 2*len(absPath))
+		n := uint32(len(out))
+		for {
+			n, err = syscall.GetLongPathName(&in[0], &out[0], n)
```

Comment:
According to this docs, the last param is:

> The size of the buffer lpszLongPath points to, in TCHARs.

Apparently TCHAR can mean char or wchar_t depending on whether your project is compiled as unicode or not. I'm not sure what Go does here, but it's possible you're telling `GetLongPathName` that it's got double the amount of space available that it actually does.